solution/zenko-version-cr.yaml

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: zenko.io/v1alpha1
+kind: ZenkoVersion
+metadata:
+  name: 1.1-dev
+spec:
+  imagePullPolicy: Always
+  versions:
+    cloudserver:
```

Comment:
There are 2 entries so that they can be decoupled :) The actual code for the 2 services is separate, and it should really be in 2 repos generating 2 images.

Reply:
alright then! thanks for the answer